tools/plugins/com.liferay.ide.hook.core/src/com/liferay/ide/hook/core/operation/NewHookDataModelProvider.java

```diff
@@ -96,7 +96,7 @@ public Object getDefaultProperty( String propertyName )
 
                         if( defaultDocroot != null )
                         {
-                            String containerFullPath = defaultDocroot.getFullPath().toPortableString();
+                            String containerFullPath = defaultDocroot.getFullPath().toPortableString() + customJspFolder;
```

Comment:
use this instaead
defaultDocroot.getFullPath().append( customJspFolder ).toPortableString()